packages/ed25519-signature-2018/src/Ed25519Signature2018.ts

```diff
@@ -238,37 +238,28 @@ export class Ed25519Signature2018 {
       throw new Error('No "verificationMethod" or "creator" found in proof.');
     }
     const { document } = await documentLoader(verificationMethod);
-    const framed = await jsonld.frame(
-      verificationMethod,
-      {
-        "@context": document["@context"],
-        "@embed": "@always",
+    const method = document.verificationMethod.find(
+      (m: any) => m.id === verificationMethod
```

Comment:
give this a type other than any, but probably it will have to support a bunch of different key types (publicKeyJwk, publicKeyBase58, publicKeyMultibase).

Reply:
Adding Type!